packages/select/src/components/omnibar/omnibar.tsx

```diff
@@ -41,6 +41,9 @@ export interface IOmnibarProps<T> extends IListItemsProps<T> {
     /** Props to spread to `Overlay`. */
     overlayProps?: Partial<IOverlayProps>;
 
+    /** The query string. */
+    query?: string;
```

Comment:
should this be `initialQuery` given that changes are not respected?

Reply:
i'm planning to change this immediately after merge with the whole un/controlled feature, so let's keep it to this name so it's not "breaking"